### PR TITLE
add endpoint to list  project's members

### PR DESCRIPTION
**Pull request by anludke**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (FLPATH-xxxx)*:
Fixes #

**Change type**
- [x] New feature
- [ ] Bug fix
- [x] Unit tests
- [ ] Integration tests
- [ ] CI
- [ ] Documentation
- [ ] Auto-generated SDK code

**Impacted services**
- [x] Workflow Service
- [ ] Notification Service

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.

### CODE REVIEW

File: workflow-service/src/main/java/com/redhat/parodos/project/service/ProjectServiceImpl.java [hunk 1]
```diff
@@ -130,7 +131,9 @@ public List<ProjectResponseDTO> getProjects() {
 				hm.get(key).add(projectUserRole);
 			}
 			else {
-				hm.put(key, List.of(projectUserRole));
+				List<ProjectUserRole> pur = new ArrayList<>();
+				pur.add(projectUserRole);
+				hm.put(key, pur);
```
Comment (gciavarrini) [inline workflow-service/src/main/java/com/redhat/parodos/project/service/ProjectServiceImpl.java:136]: nit: 
I think these lines can be `hm.put(key, List.of(projectUserRole));`

File: workflow-service/src/main/java/com/redhat/parodos/project/service/ProjectServiceImpl.java [hunk 2]
```diff
@@ -172,8 +175,9 @@ public ProjectUserRoleResponseDTO updateUserRolesToProject(UUID projectId,
 						.project(project).user(userRoleMap.getKey()).role(userRoleMap.getValue().get()).build())
 				.collect(Collectors.toSet());
 
-		projectUserRoleRepository.deleteAllByIdProjectIdAndIdUserIdIn(project.getId(),
-				projectUserRoles.stream().map(projectUserRole -> projectUserRole.getUser().getId()).toList());
+		// projectUserRoleRepository.deleteAllByIdProjectIdAndIdUserIdIn(project.getId(),
+		// projectUserRoles.stream().map(projectUserRole ->
+		// projectUserRole.getUser().getId()).toList());
```
Comment (gciavarrini) [inline workflow-service/src/main/java/com/redhat/parodos/project/service/ProjectServiceImpl.java:180]: is this comment needed?

File: workflow-service/generated/openapi/openapi.json [hunk 3]
```diff
@@ -492,7 +492,75 @@
...
+            "type" : "string",
+            "format" : "uuid"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/ProjectResponseDTO"
```
Comment (gciavarrini) [inline workflow-service/generated/openapi/openapi.json:516]: is this ok? I'm not sure if `ProjectResponseDTO` is the right type.

File: workflow-service/src/test/java/com/redhat/parodos/project/controller/ProjectControllerTest.java [hunk 4]
```diff
@@ -289,11 +298,65 @@ public void testCreateAccessRequestToProjectWhenUserAlreadyAssignedToProject() t
...
 		responseDTO.setName(name);
 		return responseDTO;
 	}
 
+	ProjectMemberResponseDTO createSampleProjectMember(String name, String... roles) {
+		ProjectMemberResponseDTO responseDTO = new ProjectMemberResponseDTO();
+		responseDTO.setUsername(name);
+		responseDTO.setFirstName(name);
+		responseDTO.setLastName(name);
+		responseDTO.setRoles(new TreeSet<>(Set.of(roles)));
```
Comment (gciavarrini) [inline workflow-service/src/test/java/com/redhat/parodos/project/controller/ProjectControllerTest.java:358]: why don't use `new TreeSet(roles)` (without the `Set.of`)?

File: workflow-service/src/main/java/com/redhat/parodos/project/service/ProjectServiceImpl.java (hunk 1 above)
Comment (RichardW98) [reply workflow-service/src/main/java/com/redhat/parodos/project/service/ProjectServiceImpl.java:136]: List.of returns immutable collection which will not be able to add item

File: workflow-service/src/test/java/com/redhat/parodos/project/controller/ProjectControllerTest.java (hunk 4 above)
Comment (RichardW98) [reply workflow-service/src/test/java/com/redhat/parodos/project/controller/ProjectControllerTest.java:358]: same reason as above List.of

**Pull request merged**